src/style.css

```diff
@@ -2,0 +1,40 @@
+.stars__body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  background-image: url(./images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars_star:nth-child(-n + 1) {
```

Comment:
Avoid using tag names for styling (except `html` and `body`). Style elements using classes instead to prevent potential conflicts and to ensure styles are not overly broad.